app/models/measure.rb

```diff
@@ -23,6 +23,7 @@ class Measure < Sequel::Model
   plugin :time_machine
   plugin :oplog, primary_key: :measure_sid, materialized: true
   plugin :national
+  plugin :optimized_many_to_many
```

Comment:
You can make this a default plugin https://github.com/trade-tariff/trade-tariff-backend/blob/HMRC-1443/config/application.rb#L52 and then you can easily roll it out everywhere